app/domain/pbs/export/tabular/people/participation_nds_row.rb

```diff
@@ -1,19 +0,0 @@
-# frozen_string_literal: true
-
-# Copyright (c) 2012-2022, Pfadibewegung Schweiz. This file is part of
-# hitobito_pbs and licensed under the Affero General Public License version 3
-# or later. See the COPYING file at the top-level directory or at
-# https://github.com/hitobito/hitobito_pbs.
-
-module Pbs::Export::Tabular::People::ParticipationNdsRow
-  extend ActiveSupport::Concern
-
-  included do
-    alias_method_chain :first_language, :correspondence_language
```

Comment:
Wir müssen diese Methode weiterhin überschreiben, weil die Base-Implementation immer 'DE' zurückgibt (Jubla hat z.B. nur Gruppen in der Deutschschweiz).